components/tensorflow-notebook-image/Dockerfile

```diff
@@ -97,7 +97,6 @@ RUN pip install --upgrade pip && \
     # Tensorflow
     ${TF_PACKAGE} \
     # Jupyter Stuff
-    jupyter \
```

Comment:
Why are you removing Jupyter; Don't we need that to run Jupyter?

Reply:
It seems `jupyter` package is an meta package, it looks quite out of date for me.

But, it is OK, if we add `jpyter-console==0.6.0`.

Let's close this PR.